### ISSUE TITLE
Android SDK 16 (Jelly Bean): Response to Permissions.request('camera') is undefined (expecting 'authorized')

### ISSUE THREAD

**Issue by sunweiyang** [zoontek/react-native-permissions#157]
My code, running on a Nexus S (Android 4.1, Jelly Bean, SDK 16) emulator:

```
const Permissions = require('react-native-permissions');

    Permissions.request('camera')
    .then(
      response => {
        console.log('Camera permission:');
        console.log(response);
      }
    )
```

I'm expecting the response to be 'authorized', but I instead see undefined.
Behavior on more recent Android versions behave as expected.

**Comment by yonahforst** [comment]
Hi @sunweiyang - I'm not sure, we are using RN's own implementation under the hood. Maybe check here: https://facebook.github.io/react-native/docs/permissionsandroid.html

**Comment by bienur** [comment]
@yonahforst the issue is that PermissionsAndroid [returns boolean](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/modules/permissions/PermissionsModule.java#L102) if android OS is less than `M`, and your code tries to always [map it to `RESULTS`](https://github.com/yonahforst/react-native-permissions/blob/master/index.android.js#L74)

**Comment by JAStanton** [comment]
Thanks for creating the PR @bienur  https://github.com/yonahforst/react-native-permissions/pull/163

**Comment by yonahforst** [comment]
resolved in #163
